synapse/lib/stormtypes.py

```diff
@@ -478,6 +478,7 @@ class Lib(StormType):
     _ismutable = False
     _storm_query = None
     _storm_typename = 'storm:lib'
+    _storm_lib_perms = ()
```

Comment:
Do we want to add a quick schema check on these when they are set, so we have consistent data?

Reply:
The only ones that can be dynamically defined currently are from power-ups. Runtime schema checking values that are hard-coded seems like overkill, but open to discussing when this would happen.